src/bp/core/telemetry/stats-service.ts

```diff
@@ -1,15 +1,15 @@
-import { TelemetryRepository } from 'core/repositories/telemetry'
```

Comment:
Shouldn't this file be inside the stats folder instead?

Reply:
I'm not sure how to see it! Which files should be found inside the stats sub-folder and which files should only be found at the root of telemetry?

Reply:
I think it make sense to keep the "most important" files at the root, and leave more specific files in a subfolder